buildSrc/src/main/rocker/com/uber/okbuck/template/config/BuckDefs.rocker.raw

```diff
@@ -10,24 +10,83 @@ boolean useCompilationClasspath,
 String classpathExclusionRegex
 )
 
-import os, re
+def basename(path):
```

Comment:
could we use skylib for this? (https://buckbuild.com/concept/skylark.html)

Reply:
I also like the idea of using skylib

Reply:
I've considered using skylib, but since we discovered that cells are still having some bugs at the moment, I'd avoid doing this, added comment describing that these methods are subset of skylib.